src/Aspire.ProjectTemplates/templates/aspire-servicedefaults/.template.config/ide.host.json

```diff
@@ -8,22 +8,13 @@
       "isVisible": false
     },
     {
-      "id": "AspireVersion",
-      "isVisible": true,
```

Comment:
Deleted isVisible on this one too. Leaving a comment in case that wasn't intentional